test/helper.js

```diff
@@ -24,5 +24,16 @@ function check (t, chunk, level, msg) {
   })
 }
 
+function check2 (t, chunk, wanted) {
+  t.ok(new Date(chunk.time) <= new Date(), 'time is greater than Date.now()')
+  delete chunk.time
+  t.deepEqual(chunk, Object.assign({
+    pid: pid,
+    hostname: hostname,
+    v: 1
+  }, wanted))
+}
+
 module.exports.sink = sink
 module.exports.check = check
+module.exports.check2 = check2
```

Comment:
That would cause code duplication, repeating the same `require`s as in `helpers.js`. This new helper allows more concise (and more flexible) coding as the original `check()` and could be used instead for new tests:
```checkGen(t, chunk, wanted)```
instead of:
```check(t, chunk, wanted.level, wanted.msg)```